src/f.jl

```diff
@@ -0,0 +1,81 @@
+# f.jl
+# F-tests
+#
+# Copyright (C) 2019   Raphael Saavedra
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+export VarianceFTest
+
+struct VarianceFTest <: HypothesisTest
+    n_x::Int   # size of sample 1
+    n_y::Int   # size of sample 2
+    df_x::Int  # degrees of freedom 1
+    df_y::Int  # degrees of freedom 2
+    F::Real    # test statistic
+    F0::Real   # variance ratio under h_0
```

Comment:
This is not used

Reply:
It's used in `show`:

```
julia> t
Variance F-test
---------------
Population details:
    parameter of interest:   variance ratio
    value under h_0:         1.0
    point estimate:          0.8073932940769317

Test summary:
    outcome with 95% confidence: fail to reject h_0
    two-sided p-value:           0.7552

Details:
    number of observations: [10, 10]
    F statistic:            0.8073932940769317
    degrees of freedom:     [9, 9]
```

Reply:
But it is not used in computing a p value... it is not providing the functionality it suggests

Reply:
I agree it does not provide any functionality. I can just remove F0 and leave value under h_0 as 1.0 as it will always be the case